metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -502,6 +503,13 @@ class MetalsLanguageServer(
       tables,
       clientConfig
     )
+    popupChoiceReset = new PopupChoiceReset(
```

Comment:
Maybe include it in the Doctor?  I feel like it's part of the Doctor's functionality.

Reply:
I always prefer smaller, single responsibility classes but I have no strong preference here, if you want I can move this logic to Doctor :+1:

Reply:
It's fine, but at some point we need a proper solution for setting up all the classes better.